src/blitter/32bpp_anim.cpp

```diff
@@ -365,7 +365,7 @@ void Blitter_32bppAnim::CopyFromBuffer(void *video, const void *src, int width,
 		Colour *dst_pal = dst;
 		uint16 *anim_pal = anim_line;
 
-		memcpy(dst, usrc, width * sizeof(uint32));
+		std::copy(usrc, usrc + width, dst);
```

Comment:
I would prefer MemCpyT

Reply:
any particular reason? std::copy basically boils down to memcpy internally, and you've recommended using copy_backward further down

Reply:
This is blitter code.
It also needs to run fast when doing unoptimised debug builds.